### PR TITLE
Remove unnecessary variance computation from backward in MVN layer

### PR DESCRIPTION
**Pull request by ducha-aiki**
Tests passed

### CODE REVIEW

**Comment by longjon** [comment]
Indeed, I also just noticed this. However, Travis is failing due to the now unnecessary `eps` in backward. (In fact, epsilon should probably be broken out into a parameter, or at the very least made a static constant belonging to the class.)

**Comment by ducha-aiki** [comment]
@longjon moved to external parameters.

**Comment by marcino239** [comment]
W11

**Comment by ducha-aiki** [comment]
@marcino239  ?

**Comment by marcino239** [comment]
Apologies,  must have been a pocket typing.

File: include/caffe/common_layers.hpp [hunk 1]
```diff
@@ -301,9 +301,9 @@ class MVNLayer : public Layer<Dtype> {
      const vector<bool>& propagate_down, const vector<Blob<Dtype>*>& bottom);
 
   Blob<Dtype> mean_, variance_, temp_;
-
   /// sum_multiplier is used to carry out sum using BLAS
   Blob<Dtype> sum_multiplier_;
+  Dtype eps;
```
Comment (jeffdonahue) [inline include/caffe/common_layers.hpp]: should be `eps_` to follow Google style guide convention for private member vars

**Comment by jeffdonahue** [comment]
LGTM besides nitpick above

**Comment by shelhamer** [comment]
Please fix nitpick + squash for merge.

**Comment by ducha-aiki** [comment]
replaced by #2465

**Pull request closed without merging**